perception/mil_vision/mil_vision_tools/labelbox_parser.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+import json
+import os
+import cv2
+import numpy as np
+
+
+class LabelBoxParser(object):
+    '''
+    Class to read an exported labelbox annotation file (must be JSON formatted)
+    and call a user specified callback function with this parsed annotation along with the
+    image it annotates.
+    '''
+    def __init__(self, labelfile, image_dir='.'):
+        '''
+        @param labelfile: name of json file containing labelbox annotations
```

Comment:
Do we have a labelbox annotation file? I imagine this really wouldn't change from data set to data set unless we plan on segmenting them by tasks (thus limiting the possible annotations). At any rate it might be a good idea to get a default json file for annotations. At some point. Not critical.

Reply:
I think you're confused about what labelfile is. This is the json file containing the exported labels from LabelBox. LabelBox also requires a template json file defining what objects can be labeled, but does tool doesn't need to know that.

Reply:
Yeah I should have updated this after I used LabelBox sorry it contains the info for the labelled images. But I was confused for sure, thought it was for the labelling itself. But that is done in the LabelBox software and is dependent on the mode you choose. Interestingly, you can extract the images from the json afterwards too, separating labels and pngs. Good stuff.